gui/builtinItemStatsViews/itemAttributes.py

```diff
@@ -111,8 +114,8 @@ def ExportItemStats(self, event):
         exportFileName = self.item.name + " (" + str(self.item.ID) + ").csv"
 
         with wx.FileDialog(
-            self, "Save CSV file", "", exportFileName,
-            "CSV files (*.csv)|*.csv", wx.FD_SAVE | wx.FD_OVERWRITE_PROMPT
+                self, _t("Save CSV file"), "", exportFileName,
+                _t("CSV files (*.csv)|*.csv"), wx.FD_SAVE | wx.FD_OVERWRITE_PROMPT
```

Comment:
For these guys, I adopted a different convention in other places, simply because the filetype shouldn't change with the translations: 

` _t("CSV files") + " (*.csv)|*.csv")`

If there's no objections with that, let switch to it to keep consistent 😊

Reply:
I'm on it. I've come up a regex for that: `['"].+\|\*\.\w+['"]`. I think `|*.xxx` part is required part for this wildcard.